node/plugin/config.go

```diff
@@ -150,6 +150,10 @@ func NewConfig(ctx *cli.Context) (*Config, error) {
 		return nil, errors.New("unsupported operation type")
 	}
 
+	if op != OperationListQuorums && len(ctx.GlobalString(EcdsaKeyFileFlag.Name)) == 0 && len(ctx.GlobalString(EcdsaKeyPasswordFlag.Name)) == 0 {
```

Comment:
nit: I'd prefer matching `op == { opt-in, opt-out, update-socket }` to `op != OperationListQuorums`. I think that's more explicit and future compatible when we add new operation?

Reply:
that's a fair point

Reply:
cca159bf8fd31d7356f648cc6908bcdae59a2ca7